internal/nonodo/nonodo.go

```diff
@@ -133,8 +143,7 @@ Press Ctrl+C to stop the node
 	} else if opts.EnableEcho {
 		fmt.Println("Starting echo app")
 		w.Workers = append(w.Workers, echoapp.EchoAppWorker{
-			RollupEndpoint: fmt.Sprintf("http://%s:%v/rollup",
-				opts.HttpAddress, opts.HttpPort),
+			RollupEndpoint: fmt.Sprintf("http://127.0.0.1:%v", DefaultRollupsPort),
```

Comment:
Changed to opts.HttpRollupsPort to use the opt instead of the default value, use the opts.HttpAddress
```
			RollupEndpoint: fmt.Sprintf("http://%s:%v/rollup",
				opts.HttpAddress, opts.HttpRollupsPort),
```

Reply:
Ok, changed
https://github.com/Calindra/nonodo/blob/2faba9a16c88b69cd33333bdaba74dd71d22a626/internal/nonodo/nonodo.go#L131